providers/zoom/zoom.go

```diff
@@ -0,0 +1,176 @@
+// Package zoom implements the OAuth2 protocol for authenticating users through zoo.
+// This package can be used as a reference implementation of an OAuth2 provider for Goth.
+package zoom
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/markbates/goth"
```

Comment:
Same here, third party imports should be grouped below separately from standard lib ones.